docs/python-sdk.md

```diff
@@ -0,0 +1,238 @@
+<!--
+title: Python SDK
+menuText: Python SDK
+description: 
+menuOrder: 6
+-->
+
+# Python SDK
+
+Serverless Console, when Instrumentation is enabled on an AWS Lambda function,
+will hook into the AWS Lambda runtime environment and automatically report
+metrics, traces, spans, and events. To capture handled errors, warnings, and to
+set custom tags, the SDK library must be added and instrumented in your AWS
+Lambda function handler.
+
+## Key terms
+
+- An **Event** is an instance of an error, warning, or notice that is captured
+as a part of a Trace. Multiple events can be captured in a single trace.
+- A **Captured Error** is a one instance of an error that is sent to Serverless
+Console as an Event. It can be viewed in Dev Mode or the Trace Explorer Details.
+- A **Captured Warning** is one instance of a string in Node.js that is sent to
```

Comment:
Refers to "Node.js", should be Python

Reply:
Suggestion: "is an instance" sounds better than "is a one instance" or "is one instance"

Reply:
Fixed